src/api.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @author acvetkov@yandex-team.ru
+ * @overview
+ */
+
+'use strict';
+
+import config from './config';
+import {wrapAsyncMethods, wrapSyncMethods} from './wrapper';
+import {get, set} from './object';
+
+/**
+ * @param {Function} Thenable
+ * @returns {ThenChrome}
+ */
+export default function create(Thenable) {
+    return Object.keys(config).reduce((result, namespace) => {
+        createNamespace(result, namespace);
```

Comment:
It will be more functional if you use that syntax: `result = createNamespace(result, namespace);` and original result will be const as well. But obviously it will be slower